### ISSUE TITLE
List - id and aria-labelledby values should be identical

### ISSUE THREAD

**Issue by davidjacob93** [manchesergit/material-ui#56]
Ensures all ARIA attributes have valid values

https://dequeuniversity.com/rules/axe/2.3/aria-valid-attr-value?application=AxeChrome

The issue here is that the id and the aria-labelled attributes of the checkbox part of the list has different values. These should be the identical to fix this issue.

**Comment by djbuckley** [comment]
@davidjacob93 please add more details to this as its not clear what the issues is
please also add the label tags needed for this issue

**Comment by davidjacob93** [comment]
This issue has been resolved by fixes on issues #55 and #15.
